ports/atmel-samd/audio_dma.c

```diff
@@ -50,6 +50,18 @@ uint8_t find_free_audio_dma_channel(void) {
     return channel;
 }
 
+void audio_dma_disable_channel(uint8_t channel) {
+    if (channel >= AUDIO_DMA_CHANNEL_COUNT)
+        mp_raise_RuntimeError(translate("Internal error: disable invalid dma channel"));
```

Comment:
I wouldn't raise this here because it is called outside of the VM potentially which will crash. Could we just rely on the enable check? Another option is an assert but I'm not actually sure that we compile them in at all.

Reply:
I went ahead and just made the function return early if the argument is out of range.